partiql-parser/src/main/antlr/PartiQLParser.g4

```diff
@@ -934,8 +941,8 @@ literal
 
 type
     : datatype=(
-        NULL | BOOL | BOOLEAN | SMALLINT | INTEGER2 | INT2 | INTEGER | INT | INTEGER4 | INT4
-        | INTEGER8 | INT8 | BIGINT | REAL | CHAR | CHARACTER | MISSING
```

Comment:
(self-review): `NULL` and `MISSING` are no longer passed as data types.

Reply:
Nice!